package.json

```diff
@@ -47,6 +48,7 @@
     "vitest": "^3.2.4"
   },
   "dependencies": {
+    "axios": "^1.12.2",
```

Comment:
can be removed for now - notice the difference in versions though - keep an eye on that going forward